src/renderer/components/+landing-page/landing-page.tsx

```diff
@@ -23,17 +23,23 @@ export class LandingPage extends React.Component {
   }
 
   componentDidMount() {
-    const workspacesSeen = new Set(LandingPage.storage.get());
+    // ignore workspaces that don't exist
+    const seenWorkspaces = new Set(
+      LandingPage
+        .storage
+        .get()
+        .filter(id => workspaceStore.getById(id))
+    );
 
-    if (!workspacesSeen.has(this.workspace.id) && this.workspaceClusters.length === 0) {
+    if (!seenWorkspaces.has(this.workspace.id) && this.workspaceClusters.length === 0 && !this.workspace.isManaged) {
```

Comment:
just noticed this might need to be done in a `reaction()`, because this page may already be visible when user creates a new one, or switches to an empty one.